package.nls.json

```diff
@@ -101,5 +106,48 @@
+  "walkthroughs.ibmiDevelopmentLearn.title": "IBM i Development",
+  "walkthroughs.ibmiDevelopmentLearn.description": "Now that your project is set up, click below to learn more about how to use the IBM i Project Explorer for development.\n[Learn More](command:vscode-ibmi-projectexplorer.openDevelopmentWalkthrough)",
+  "walkthroughs.ibmiDevelopmentLearn.altText": "Development with the IBM i Project Explorer",
+  "walkthroughs.ibmiDevelopment.title": "Development with an Existing IBM i Project",
+  "walkthroughs.ibmiDevelopment.description": "Learn about development using the IBM i Project Explorer",
+  "walkthroughs.cloneRepository.title": "Clone Repository",
+  "walkthroughs.cloneRepository.description": "If you do not already have your IBM i project in the workspace, clone the repository into the current workspace.\n[Clone Repository](command:git.clone)",
+  "walkthroughs.cloneRepository.altText": "Clone repository into workspace",
+  "walkthroughs.deployProject.title": "Deploy Project",
+  "walkthroughs.deployProject.description": "To visualize, compare, and deploy your local source files, set the deploy location for the project using the inline action on the Source heading.\n[Set Deploy Location](command:vscode-ibmi-projectexplorer.setDeployLocation)\nYou will also need to set the project's deployment method which impacts which files will be deployed.\n[Set Deployment Method](command:vscode-ibmi-projectexplorer.projectExplorer.setDeploymentMethod)\nBegin the deployment process by using the Deploy Project action.\n[Deploy Project](command:vscode-ibmi-projectexplorer.projectExplorer.deployProject)",
+  "walkthroughs.deployProject.altText": "Deploy Project action",
+  "walkthroughs.buildEntireProject.title": "Build Entire Project",
+  "walkthroughs.buildEntireProject.description": "Before building the entire project, right-click on the project and set the build command to be used based on the desired build tool such as elias in ARCAD Builder or makei in ibmi-bob.\n[Set Build Command](command:vscode-ibmi-projectexplorer.projectExplorer.setBuildCommand)\nRunning a build will first deploy the project and then invoke the build command.\n[Run Build](command:vscode-ibmi-projectexplorer.projectExplorer.runBuild)",
+  "walkthroughs.buildEntireProject.altText": "Run Build action",
+  "walkthroughs.compileIndividualFiles.title": "Compile Individual Files",
+  "walkthroughs.compileIndividualFiles.description": "Similar to the build command, the project's compile command must be set first before compiling any files.\n[Set Compile Command](command:vscode-ibmi-projectexplorer.projectExplorer.setCompileCommand)\nTo compile the active text editor in which you are working in, using the Run Compile action located at the top right of the editor or in the VS Code command palette.\nTo compile an entire directory or a specific file, use the right-click Run Compile action on any file or directory in the File Explorer or under the Source heading.",
```

Comment:
To compile the file in the active text editor in whcih ... command palette.  This can be done even more conveniently using the keyboard shortcut (.. on Mac or Ctrl-Shift-C on Windows).